Makefile.am

```diff
@@ -13,7 +13,7 @@ AM_CFLAGS = $(EXTRA_CFLAGS) \
     $(TSS2_SYS_CFLAGS) $(TCTI_DEVICE_CFLAGS) $(TCTI_SOCKET_CFLAGS) \
     $(CODE_COVERAGE_CFLAGS)
 AM_LDFLAGS = $(EXTRA_LDFLAGS) $(CODE_COVERAGE_LIBS)
-UNIT_AM_CFLAGS = $(AM_CFLAGS) $(CMOCKA_CFLAGS)
+UNIT_AM_CFLAGS = $(AM_CFLAGS) $(CMOCKA_CFLAGS) -Wno-unused-parameter
```

Comment:
I added it specifically to the files where there are functions with unused parameters.  Usually because of a fixed prototype.
An alternative is to disable it globally.

Per our chat session, I will add a macro to the functions where it appears to force the parameter to be used.  Something like:
#define UNUSED_PARAM(p) (void)(p);